configuration/concepts/orders_obstetricsGynecology.csv

```diff
@@ -1,16 +1,16 @@
 c9c2baca-06ee-4975-a2bc-f400f99c6ed4,,,OBG1005 - Annexectomie,OBG1005 - Annexectomie,Annexectomie,Annexectomie,,,Obstetrics and Gynecology,Text,,,,,,,,,,,
 af2dbc02-6e0f-4552-a64f-50aeafc0ff53,,,OBG1006 - Biopsie cervicale,OBG1006 - Biopsie cervicale,Biopsie cervicale,Biopsie cervicale,,,Obstetrics and Gynecology,Text,,,,,,,,,,,
 badfcdc6-3993-4d0a-af97-5c01d380bad9,,,OBG1007 - Biopsie endometriale,OBG1007 - Biopsie endometriale,Biopsie endometriale,Biopsie endometriale,,,Obstetrics and Gynecology,Text,,,,,,,,,,,
 f643eccf-d435-4db6-a35d-85cc4d749657,,,OBG1008 - Cerclage,OBG1008 - Cerclage,Cerclage,Cerclage,,,Obstetrics and Gynecology,Text,,,,,,,,,,,
-f16675de-d6bb-4fd1-914c-a25d8cf59465,,,OBG1009 - Césarienne (suivi à l'HSC),OBG1009 - Césarienne (suivi à l'HSC),Césarienne (suivi à l'HSC),Césarienne (suivi à l'HSC),,,Obstetrics and Gynecology,Text,,,,,,,,,,,
-c2510453-994d-4a49-9016-a77e883382c0,,,OBG1010 - Césarienne (suivi hors de l'HSC),OBG1010 - Césarienne (suivi hors de l'HSC),Césarienne (suivi hors de l'HSC),Césarienne (suivi hors de l'HSC),,,Obstetrics and Gynecology,Text,,,,,,,,,,,
-3517d312-8a22-4f60-a098-e9cc0bb5a49b,,,OBG1011 - Césarienne + Hysterectomie (suivi à l'HSC),OBG1011 - Césarienne + Hysterectomie (suivi à l'HSC),Césarienne + Hysterectomie (suivi à l'HSC),Césarienne + Hysterectomie (suivi à l'HSC),,,Obstetrics and Gynecology,Text,,,,,,,,,,,
-865bb6bf-b38b-4a6d-a013-834e450e26d5,,,OBG1012 - Césarienne + Hysterectomie (suivi hors l'HSC),OBG1012 - Césarienne + Hysterectomie (suivi hors l'HSC),Césarienne + Hysterectomie (suivi hors l'HSC),Césarienne + Hysterectomie (suivi hors l'HSC),,,Obstetrics and Gynecology,Text,,,,,,,,,,,
+f16675de-d6bb-4fd1-914c-a25d8cf59465,,,OBG1009 - Césarienne (suivi a la clinique),OBG1009 - Césarienne (suivi a la clinique),Césarienne (suivi a la clinique),Césarienne (suivi a la clinique),,,Obstetrics and Gynecology,Text,,,,,,,,,,,
+c2510453-994d-4a49-9016-a77e883382c0,,,OBG1010 - Césarienne (suivi hors de la clinique),OBG1010 - Césarienne (suivi hors de la clinique),Césarienne (suivi hors de la clinique),Césarienne (suivi hors de la clinique),,,Obstetrics and Gynecology,Text,,,,,,,,,,,
+3517d312-8a22-4f60-a098-e9cc0bb5a49b,,,OBG1011 - Césarienne + Hysterectomie (suivi a la clinique),OBG1011 - Césarienne + Hysterectomie (suivi a la clinique),Césarienne + Hysterectomie (suivi a la clinique),Césarienne + Hysterectomie (suivi a la clinique),,,Obstetrics and Gynecology,Text,,,,,,,,,,,
+865bb6bf-b38b-4a6d-a013-834e450e26d5,,,OBG1012 - Césarienne + Hysterectomie (suivi hors la clinique),OBG1012 - Césarienne + Hysterectomie (suivi hors la clinique),Césarienne + Hysterectomie (suivi hors la clinique),Césarienne + Hysterectomie (suivi hors la clinique),,,Obstetrics and Gynecology,Text,,,,,,,,,,,

```

Comment:
@nuhuhmutebi I seem to still see accents **Césarienne** which should be **Cesarienne**